products/run-run-and-sweep-github-files/faq.md

```diff
@@ -4,6 +4,23 @@ description: Frequently asked questions about Runs
 
 # FAQ
 
+## How do I retain my run data?
+The downloadable artifacts are how you get data out of the runs.  For IxNodes, there are a few options:
+1. You can download straight from JupyterHub
+2. You can mount the data locally with `grid interactive mount`
+3. You can `scp`
+4. You can simply 'pause' the node and restart it any time you need it (if you don't require downloading the data locally)
+
+## Where are artifacts stored?
```

Comment:
**Where are artifacts stored?**
Why do users need to know where they're stored? I agree that we should let them know how long artifacts are available. Maybe create a separate FAQ for that.